jupyter-on-gke/jupyter_config/config-selfauth.yaml

```diff
@@ -23,7 +23,7 @@
 # Available chart versions: https://jupyterhub.github.io/helm-chart/
 hub:
   image: 
-    name: <Repo name here>
+    name: us-docker.pkg.dev/ai-on-gke/jupyterhub-authentication-class/jupyter-auth-class
```

Comment:
Can we pull this out to a separate PR and submit it sooner? It's not related to GCS right?

Reply:
Created a separate PR: https://github.com/GoogleCloudPlatform/ai-on-gke/pull/133